observatory-platform/observatory/platform/telescopes/snapshot_telescope.py

```diff
@@ -54,19 +59,28 @@ def __init__(self, dag_id: str, start_date: datetime, schedule_interval: str, da
         :param queue: the Airflow queue name.
         :param max_retries: the number of times to retry each task.
         :param max_active_runs: the maximum number of DAG runs that can be run at once.
+        :param source_format: the prefix used to find the schema path
```

Comment:
Description of schema_prefix instead of source_format

Reply:
Thanks, I've fixed that now.